lib/shopify-cli/commands/connect.rb

```diff
@@ -3,99 +3,31 @@
 module ShopifyCli
   module Commands
     class Connect < ShopifyCli::Command
+      prerequisite_task :regenerate_env
+
       def call(*)
         project_type = ask_project_type unless Project.has_current?
-
-        if Project.has_current? && Project.current && Project.current.env
-          @ctx.puts @ctx.message('core.connect.already_connected_warning')
-          prod_warning = @ctx.message('core.connect.production_warning')
-          @ctx.puts prod_warning if [:rails, :node].include?(Project.current_project_type)
-        end
-
-        env_data = begin
-          Resources::EnvFile.parse_external_env
-                   rescue Errno::ENOENT
-                     {}
-        end
-
+        @api_key = Project.current.env['api_key']
         org = fetch_org
-        id = org['id']
-        app = get_app(id, org['apps'])
-        shop = get_shop(org['stores'], id)
-
-        write_env(app, shop, env_data[:scopes], env_data[:extra])
-        write_cli_yml(project_type, id) unless Project.has_current?
-
-        @ctx.puts(@ctx.message('core.connect.connected', app['title']))
-      end
-
-      def ask_project_type
-        CLI::UI::Prompt.ask(@ctx.message('core.connect.project_type_select')) do |handler|
-          ShopifyCli::Commands::Create.all_visible_type.each do |type|
-            handler.option(type.project_name) { type.project_type }
-          end
-        end
+        write_cli_yml(project_type, org['id']) unless Project.has_current?
```

Comment:
I don't think it is at all right to write a value to an env file to support our cli code and all of our current code extracts the org ID from the cli file so it would be in the wrong place. I would recommend returning the value.

Reply:
A little bit more confusion from my end, the next line (`@ctx.puts(@ctx.message('core.connect.connected', app['title']))`)  prints the app title associated with this and I'm confused about how to get that without the org itself. Is there another way to get this information with just the api_key?